test/rules/no-unnecessary-qualifier/test-global.ts.lint

```diff
@@ -1,6 +1,6 @@
-namespace N {
+namespace Global {
```

Comment:
Due to changes to the test runner this is now compiled along with the other files in this directory -- which already declares `namespace N`.